permits/migrations/0055_sort_contact_list.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.7 on 2021-12-17 14:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('permits', '0054_add_help_text_for_validation_comment_fields'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='historicalpermitrequest',
+            name='creditor_type',
+            field=models.PositiveSmallIntegerField(blank=True, choices=[(7, 'Architecte/Ingénieur'), (6, 'Association'), (0, 'Autres'), (8, 'Direction des travaux'), (3, 'Entreprise'), (4, "Maître d'ouvrage"), (2, 'Propriétaire'), (1, "Requérant (si différent de l'auteur de la demande)"), (5, 'Sécurité')], null=True, verbose_name='Destinaire de la facture'),
+        ),
+        migrations.AlterField(
+            model_name='permitactortype',
+            name='type',
+            field=models.PositiveSmallIntegerField(choices=[(7, 'Architecte/Ingénieur'), (6, 'Association'), (0, 'Autres'), (8, 'Direction des travaux'), (3, 'Entreprise'), (4, "Maître d'ouvrage"), (2, 'Propriétaire'), (1, "Requérant (si différent de l'auteur de la demande)"), (5, 'Sécurité')], default=0, verbose_name='type de contact'),
+        ),
+        migrations.AlterField(
+            model_name='permitrequest',
+            name='creditor_type',
+            field=models.PositiveSmallIntegerField(blank=True, choices=[(7, 'Architecte/Ingénieur'), (6, 'Association'), (0, 'Autres'), (8, 'Direction des travaux'), (3, 'Entreprise'), (4, "Maître d'ouvrage"), (2, 'Propriétaire'), (1, "Requérant (si différent de l'auteur de la demande)"), (5, 'Sécurité')], null=True, verbose_name='Destinaire de la facture'),
+        ),
+        migrations.AlterField(
+            model_name='permitrequestactor',
+            name='actor_type',
+            field=models.PositiveSmallIntegerField(choices=[(7, 'Architecte/Ingénieur'), (6, 'Association'), (0, 'Autres'), (8, 'Direction des travaux'), (3, 'Entreprise'), (4, "Maître d'ouvrage"), (2, 'Propriétaire'), (1, "Requérant (si différent de l'auteur de la demande)"), (5, 'Sécurité')], default=0, verbose_name='type de contact'),
+        ),
+    ]
```

Comment:
Please squash migrations

Reply:
Ok, will do it after addressing your comments.

What's the command?